backend/main.py

```diff
@@ -113,6 +113,36 @@ def new_game():
   return jsonify(firebase.put('/games', game, put_data))
 
 
+@app.route('/joinGame', methods=['POST'])
+def join_game():
+  request_data = request.get_json()
+  game = request_data['gameId']
+  player = request_data['playerId']
+  name = request_data.get('name', '')
+  need_gun = request_data.get('needGun', False)
+  profile_image_url = request_data.get('profileImageUrl', '')
+  start_as_zombie = request_data.get('startAsZombie', False)
+  volunteer = request_data.get('volunteer', False)
+  be_secret_zombie = request_data.get('beSecretZombie', False)
+  user_id = request.headers['userId']
```

Comment:
That was taken from the spec; userId was not to be included in the info passed as part of the JSON data (according to that, normally, playerId would be taken from the header as well but should be retrieved from the JSON for exclusively this case).